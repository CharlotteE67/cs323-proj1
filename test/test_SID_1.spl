struct test{

    int valid_int,valid_hexint;
    char valid_hexchar = '\x0',valid_char = '?';
    char invalid_char = 'izfr';

};
int main(){
    int invalid_int_1 = 0295;
    int invalid_int_2 = 5920483965;

    int invalid_hexint_1 = 0x00242;
    int invalid_hexint_2 = 0xadg1u3;
    int invalid_hexint_3 = 0x123456789a;

    struct test test_struct;

    test_struct.valid_int = 24235;
    test_struct.valid_hexint = 0xabcdef;


}