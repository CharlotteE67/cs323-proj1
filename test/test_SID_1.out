Error type A at Line 5: too many chars
Error type A at Line 14: number starts with 0
Error type A at Line 15: int number overflow
Error type A at Line 17: hexadecimal number starts with redundant 0
Error type A at Line 18: hexadecimal number with illegal char
Error type A at Line 19: int number overflow